lib/public/views/Runs/ActiveColumns/runDetectorsAsyncQcActiveColumns.js

```diff
@@ -66,17 +74,32 @@ const runDetectorAsyncQualityDisplay = ({ dataPassId, simulationPassId }, run, d
 
     let qcSummaryDisplay = null;
     if (!badEffectiveRunCoverage) {
-        if (explicitlyNotBadEffectiveRunCoverage === 1) {
-            qcSummaryDisplay = getQcSummaryDisplayWithColor(notBadPercentageFormat, { color: QC_SUMMARY_COLORS.ALL_GOOD });
-        } else {
-            qcSummaryDisplay = getQcSummaryDisplayWithColor(notBadPercentageFormat, { color: QC_SUMMARY_COLORS.PARTIALLY_GOOD_NOT_BAD });
-        }
+        qcSummaryDisplay = getQcSummaryDisplayWithColor(
+            nonMcReproducibleQcSummaryContent,
+            {
+                color: explicitlyNotBadEffectiveRunCoverage === FULL_COVERAGE
+                    ? QC_SUMMARY_COLORS.ALL_GOOD
+                    : QC_SUMMARY_COLORS.PARTIALLY_GOOD_NOT_BAD,
+            },
+        );
+    } else if (!explicitlyNotBadEffectiveRunCoverage && mcReproducible) {
+        qcSummaryDisplay = getQcSummaryDisplayWithColor(
+            [
+                notBadPercentageFormat,
+                h('.d-inline-block.va-top.f7.italic', 'MC.R'),
```

Comment:
why not using `em` if what you need is emphasis?

Reply:
changed